README.md

```diff
@@ -29,7 +29,7 @@ A [MongoDB](https://www.mongodb.com) database is optional, but required for many
 
 ## Install
 
-Learn how to [set up an Indiekit server](https://getindiekit.com/get-started) and view an [example server configuration](https://github.com/paulrobertlloyd/paulrobertlloyd-indiekit/blob/main/index.js).
+Learn how to [set up an Indiekit server](https://getindiekit.com/get-started) and view an [example server configuration](https://github.com/paulrobertlloyd/paulrobertlloyd-indiekit/blob/main/indiekit.config.cjs).
```

Comment:
Oh, good catch! I’ve actually updated the example configuration provided by this project, which gives a bit more context and is less specific to my own needs, so should probably link to that now instead:

```suggestion
Learn how to [set up an Indiekit server](https://getindiekit.com/get-started) and view an [example server configuration](https://github.com/getindiekit/example-config).
```